main/networking/Runtime.ts

```diff
@@ -77,27 +78,160 @@ function createPacket(payload: unknown, messageType: MsgType): Buffer {
     case MsgType.CHALLENGE_DATA:
       encodedPayload = protos.Text.encode(payload as protos.IText).finish();
       break;
+    case MsgType.INPUTS:
+      encodedPayload = protos.Text.encode(payload as any).finish();
```

Comment:
You shouldn't re-encode it. You can just do `encodedPayload = payload;` and it should work since it already comes encoded from the UDP connection.

Reply:
Oh yea oops this was a mistake, thanks for the catch

Reply:
Fixed